modulemd/v2/modulemd-module.c

```diff
@@ -235,7 +324,24 @@ modulemd_module_add_stream (ModulemdModule *self, ModulemdModuleStream *stream)
       old = NULL;
     }
 
-  newstream = modulemd_module_stream_copy (stream, NULL, NULL);
+  if (modulemd_module_stream_get_mdversion (stream) < index_mdversion)
```

Comment:
Same as `Defaults`: the `modulemd_module_stream_upgrade` function already returns a copy if `current_mdversion == mdversion`, so maybe just use that fact here?

Reply:
Same as above.